schemas/Composer.json

```diff
@@ -20,8 +20,12 @@
     "startsThread": {
       "description": "Whether this message begins a new thread or not.",
       "type": "boolean"
+    },
+    "isRichTextEditor": {
+      "description": "Whether this message was composed in the WYSIWYG-style rich text editor.",
```

Comment:
Given there are several modes (rich text with formatting enabled; plain text with markdown enabled; and plain text with markdown disabled), could it be worth capturing this info too?

Reply:
Just a thought: an alternative to `isPlainTextMode` could be `isRichTextEditorFormattingEnabled` which both implies the association with the RTE and matches the UI toggle name.

But I don't see it being a big problem.